device/globals/telon.tscn

```diff
@@ -106,7 +106,7 @@ light_on_map = false
 tooltip = ""
 action = ""
 events_path = ""
-global_id = ""
+global_id = "telon"
```

Comment:
Should be safe to remove this from `globals/main.tscn` now, right?

Reply:
Tested removing it, force-pushing the removal, because it works.